### PR TITLE
Elixir workshops banner

### PR DESCRIPTION
**Pull request by marcoow**
This adds a banner for the Elixir workshops to the blog layout that will appear above all posts (Ember stays below all posts).

### CODE REVIEW

File: _includes/elixir_workshop_banner.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="secondary-banner workshop-banner elixir-workshop-banner">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-10 col-md-offset-1 col-sm-6 col-xs-12">
+        <a href="https://elixir-phoenix-workshops.simplabs.com">Learn Elixir and Phoenix at our Workshops!</a>
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (ganeshbsub) [inline _includes/elixir_workshop_banner.html]: Newline missing at EOF.

File: _includes/ember_workshop_banner.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="secondary-banner workshop-banner ember-workshop-banner">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-10 col-md-offset-1 col-sm-6 col-xs-12">
+        <a href="https://ember-workshops.simplabs.com">Join our Workshops to advance your Ember.js skills!</a>
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (ganeshbsub) [inline _includes/ember_workshop_banner.html]: Newline missing at EOF.

File: css/app.scss [hunk 3]
```diff
@@ -141,6 +141,10 @@ ul {
     }
   }
 
+  &.elixir-workshop-banner {
+    margin-top: 60px;
```
Comment (ganeshbsub) [inline css/app.scss:145]: Do you think the spacing appears to be too large? Or is this okay?

**Review by ganeshbsub — CHANGES_REQUESTED** [review]
Just some missing newline at EOF and I left one comment about the top margin being too large.

Other than that everything looks fine 👍

**Comment by marcoow** [comment]
Can you fix the newlines and merge? The margin was intentional as that is equal to the top margin of the main content, thus effectively vertically centering the banner between its surrounding content elements.

**Review by ganeshbsub — APPROVED** [review]

**Pull request merged**